Examples/Objective-C/Podfile.lock

```diff
@@ -43,7 +43,7 @@ PODS:
   - XLData/RemoteDataStore (2.0.0):
     - XLData/CoreRemote
     - XLData/DataStore
-  - XLForm (3.1.2)
+  - XLForm (3.3.0)
```

Comment:
Seems that Example project is actually using XLForm 3.3.0 instead of 4.0? @santiagofm